pkg/gotohelm/transpiler.go

```diff
@@ -858,15 +869,28 @@ func (t *Transpiler) transpileCallExpr(n *ast.CallExpr) Node {
 	}
 
 	// Method call.
-	if callee.Type().(*types.Signature).Recv() != nil {
+	if r := callee.Type().(*types.Signature).Recv(); r != nil {
```

Comment:
I think it might be better to move this below the really big switch statement as that would let us more easily special case `Values.AsMap`.

In the numerics PR (I think) there's a change that turns the `id` variable into something like: `github.com/redpanda-data/helm-charts/pkg/gotohelm/helmette.(*Values).AsMap`. That (roughly) matches the way that `reflect` shows functions as strings and is preferable IMO. WDYT?

Reply:
> That (roughly) matches the way that reflect shows functions as strings and is preferable IMO. WDYT?

I agree. Should I wait to rebase on top of https://github.com/redpanda-data/helm-charts/pull/1332?